card/config/initializers/cypress_dev.rb

```diff
@@ -0,0 +1,9 @@
+if defined?(CypressDev)
```

Comment:
Something I need to install?

Reply:
I don't think so. You only need to know the magic command that will let you enter the testing heaven.